src/Explorer/Controls/TreeComponent/treeComponent.less

```diff
@@ -3,11 +3,12 @@
 .treeComponent {
   .nodeItem {
     &:focus {
-      outline: 1px dashed @AccentMedium;
+      outline: 2px @AccentMedium;
     }
 
     .treeNodeHeader {
-      padding: @SmallSpace 2px;
+      // padding: @SmallSpace 2px;
+      margin: 2px;
```

Comment:
It looks like this shrinks the height of the tree node header: https://ms.portal.azure.com/?dataExplorerSource=https%3A%2F%2Fcosmos-explorer-preview.azurewebsites.net%2Fcommit%2Ff27706bc3769d225daa72c2edb1a92552fb5bb60%2Fexplorer.html%3Ffeature.pr%3Dhttps%253A%252F%252Fgithub.com%252FAzure%252Fcosmos-explorer%252Fpull%252F1404%2523sampath-accessibility-sev-2-2#@microsoft.onmicrosoft.com/resource/subscriptions/220fc532-6091-423c-8ba0-66c2397d591b/resourceGroups/vimeng-rg/providers/Microsoft.DocumentDb/databaseAccounts/vimeng-sql-test/dataExplorer.

Is there a particular reason why we need to switch from padding to margin?

Reply:
When a user uses keyboard, the outline around the tree-item is not visible entirely while we use padding, but with margin the outline will be visible which is a requirement for the accessibility.